packages/js/core/src/types/CoreClient.ts

```diff
@@ -60,22 +67,22 @@ export interface CoreClient extends Invoker, UriResolverHandler<unknown> {
    *
    * @returns an array of interfaces and their registered implementations
    */
-  getInterfaces(): readonly InterfaceImplementations[] | undefined;
```

Comment:
I noticed some inconsistencies in this file when it comes to `readonly` and `ReadOnly<...>` usage. Could we please use it the same in all methods / properties? If I'm missing some nuances please inform :)